must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -205,14 +205,20 @@ for ns in $namespaces; do
 
     oc delete -f pod_helper.yaml
 
+    crash_collection(){
+        oc debug nodes/"${node}" --to-namespace="${ns}" -- bash -c "sleep 5m" &
+        sleep 60
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}"| grep "${node//./}"-debug| awk '{print $1}')":/host/var/lib/rook/openshift-storage/crash/ "${CRASH_OUTPUT_DIR}"
+        wait
```

Comment:
I believe this wait would wait the full five minutes for the sleep to finish even if the rsync finishes. Is this what we want?

Reply:
yes because otherwise it debug pods will be running in bg after MG completes it's run.

Reply:
Why do we have to wait? Just remove the `wait` and let the `oc debug` command timeout on its own. It's not going to hurt anything by staying around. We must be efficient about gathering logs from every node, and can't afford to sleep when connecting to many nodes.

Reply:
Why not just `wait` for the `rsync` to be done since they run in // and get rid of the `sleep 5m`?
So we exit as soon as we have finished rsyncing logs.

Reply:
oc debug pod needs to be in // because it rsh into the pod when run it.

Reply:
there must be a better way! maybe the polling every few seconds that travis mentioned in a different comment. or some real timeout wrapper from coreutils. but the 5m sleep does not scale. :-)

Reply:
That's for the `sleep 60` timer. Here we just want to get rid of the wait, since we want to exit EITHER when the debug Pod times out and disappears or the rsync finishes, whichever comes first. The `sleep 5m` above is to keep the debug Pod alive, since it will just immediately exit and terminate once the provide command finishes.

Reply:
removed the wait